### ISSUE TITLE
Creating leaderboard with neg_mean_squared_error returns negative scores not sorted properly

### ISSUE THREAD

**Issue by derekflint** [deeplearninc/auger-ml#38]
It looks like -0.01 is shown to be greater than -0.00000000034

**Comment by derekflint** [comment]
@skatedplrn i'm reading sklearn docs. http://scikit-learn.org/stable/modules/model_evaluation.html

It looks like:

3.3.1.2. Defining your scoring strategy from metric functions

The module sklearn.metrics also exposes a set of simple functions measuring a prediction error given ground truth and prediction:
functions ending with _score return a value to maximize, the higher the better.
functions ending with _error or _loss return a value to minimize, the lower the better. When converting into a scorer object using make_scorer, set the greater_is_better parameter to False (True by default; see the parameter description below).
